File: go/chapter10/common_words.go
package main

import (
	"fmt"
	"os"
	"strings"
)

func main() {
	fileBytes, err := os.ReadFile("./romeo_juliet.txt")
	if err != nil {
		panic(err)
	}

	count := strings.Count(strings.ToLower(string(fileBytes)), strings.ToLower("Juliet"))
	fmt.Println(count)
}

File: javascript/chapter10/common_words.js
import fs from "fs";

try {
	const fileContents = fs.readFileSync("./romeo_juliet.txt");
	const count = String(fileContents).split("Juliet").length - 1;
	console.log(count);
} catch (e) {
	console.log(e.message);
}
